src/Controls/src/Core/Internals/MaybeNullWhenAttribute.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// This was copied from https://github.com/dotnet/runtime/blob/39b9607807f29e48cae4652cd74735182b31182e/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs
+// and updated to have the scope of the attributes be internal.
+
+#nullable disable
+
+namespace System.Diagnostics.CodeAnalysis
+{
+#if !NETCOREAPP
+
+	/// <summary>Specifies that null is allowed as an input even if the corresponding type disallows it.</summary>
+	[AttributeUsage(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property, Inherited = false)]
+	internal sealed class AllowNullAttribute : Attribute { }
+
+	/// <summary>Specifies that null is disallowed as an input even if the corresponding type allows it.</summary>
+	[AttributeUsage(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property, Inherited = false)]
+	internal sealed class DisallowNullAttribute : Attribute { }
+
+	/// <summary>Specifies that an output may be null even if the corresponding type disallows it.</summary>
+	[AttributeUsage(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property | AttributeTargets.ReturnValue, Inherited = false)]
+	internal sealed class MaybeNullAttribute : Attribute { }
```

Comment:
I got this from:

https://source.dot.net/#Microsoft.Build.Framework/NullableAttributes.cs,68093cc4b5713519

So we can use these NRT annotations in a `netstandard2.0` library.

Reply:
We did bad things in essentials:

```cs
#if !NETSTANDARD
		[return: NotNullIfNotNull("defaultValue")]
#endif
```

One day this can all go away and we use net6/7 everywhere.